src/profiling/with_rustc_perf.md

```diff
@@ -0,0 +1,35 @@
+# Profiling with rustc-perf
+
+The [Rust benchmark suite][rustc-perf] provides a comprehensive way of profiling and benchmarking
+the Rust compiler. You can find instructions on how to use the suite in its [manual][rustc-perf-readme].
+
+However, using the suite manually can be a bit cumbersome. To make this easier for `rustc` contributors,
+the compiler build system (`bootstrap`) also provides built-in integration with the benchmarking suite,
+which will download and build the suite for you, build a local compiler toolchain and let you profile or
+benchmark it using a simplified command-line interface.
+
+You can use the `./x perf -- <command> [options]` command to use this integration.
```

Comment:
Can you give a complete example of a command that someone can run that roughly matches rustc-perf does? I.e. if I want to mimic what `rust-timer` does, what should I run?

Reply:
The command also doesn't seem to be working for me. I tried `x perf -- eprintln --profiles Check`, but after it ran, all the files in `build/tmp/rustc-perf/results` were completely empty.

Reply:
The compiler doesn't normally print anything to stderr. Usually you use eprintln by adding manual eprintln calls, and then postprocess the results (see rustc-perf documentation for more info).

Currently, this can't really do what rust-timer does, as benchmarking is not supported yet (only profiling).

Reply:
Ah, perhaps you could add a clarification like this to the docs?

Reply:
Pushed a new commit that tries to make this a bit more explicit.